ng-dev/release/versioning/version-branches.ts

```diff
@@ -60,33 +69,28 @@ export async function getVersionOfBranch(
   if (!content) {
     throw Error(`Unable to read "package.json" file from repository.`);
   }
-  const {version} = JSON.parse(Buffer.from(content, 'base64').toString()) as {
+  const {version, __ngDevExceptionalMinor__} = JSON.parse(
```

Comment:
This deconstruction works because we define the value for `exceptionalMinorPackageIndicator` on line 43, but if we were to change it there this deconstruction fails right?

Should we just put the string directly in the key for the object?

Reply:
oops, very good catch. that's not supposed to be like this